openedx/core/djangoapps/theming/storage.py

```diff
@@ -192,51 +192,56 @@ def converter(matchobj):
             This requires figuring out which files the matched URL resolves
             to and calling the url() method of the storage.
             """
-            matched, url = matchobj.groups()
+            matches = matchobj.groupdict()
```

Comment:
This code is from django. They have made some changes in `django4.2`. So updated this code also. Otherwise it gives errors with collectstatic command.

Reply:
```
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/django/contrib/staticfiles/storage.py", line 301, in post_process
    for name, hashed_name, processed, _ in self._post_process(
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/django/contrib/staticfiles/storage.py", line 372, in _post_process
    content = pattern.sub(converter, content)
  File "/edx/app/edxapp/edx-platform/openedx/core/djangoapps/theming/storage.py", line 239, in converter
    return template % unquote(transformed_url)
TypeError: format requires a mapping
```